apps/crn-frontend/src/network/teams/api.ts

```diff
@@ -117,9 +117,10 @@ export const updateTeamResearchOutput = async (
   researchOutputId: string,
   researchOutput: ResearchOutputPostRequest,
   authorization: string,
+  publish: boolean = false,
 ): Promise<ResearchOutputResponse> => {
   const resp = await fetch(
-    `${API_BASE_URL}/research-outputs/${researchOutputId}`,
+    `${API_BASE_URL}/research-outputs/${researchOutputId}?publish=${publish}`,
```

Comment:
It is weird that this is a get parameter. Why wouldn't it just be setting `published` to true in the body?

Reply:
to be fair i thought the same when i saw this - why not just use the `published` property of the ResearchOutput? Or whatever it is that you would use to distinguish the published research from a draft one.